pkg/utils/builders.go

```diff
@@ -54,6 +55,15 @@ func getEnvFromMap(env map[string]string) []corev1.EnvVar {
 		perEnv.Value = v
 		envVar = append(envVar, perEnv)
 	}
+	// Getting pod name from downward api
+	PodName := GetValueFromDownwardAPI("v1", "metadata.name")
```

Comment:
Can we change this to experimentPodName to avoid the confusion that Rahul is also alluding to?